micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspect.groovy

```diff
@@ -41,7 +51,7 @@ class MetricsAspect {
         final URI uri = new URI(uriString)
         final String path = fixSpecialCharacters(uri.path)
         int port = (uri.port > 0)? uri.port : 80
-        return trimDots("${PREFIX}.${uri.host}.${port}.${path}")
+        return sanatizeIpAddressIfPresent(trimDots("${PREFIX}.${uri.host}.${port}.${path}"))
```

Comment:
@wybczu You are right, my bad. In that case the tests should look as follows:

	'http://192.168.1.100:6006'              || '192_168_1_100.6006'
	'http://192.168.1.100:6006/ticket/pay'   || '192_168_1_100.6006.ticket.pay'
	'http://ws.4finance.net:6006/ticket/pay' || 'ws_4finance_net.6006.ticket.pay'

Is that correct?